scripts/create-a-m-o/templates/template-package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "__",
+  "version": "__",
+  "description": "> TODO: description",
```

Comment:
What happened to name and version? I and imho we should provide a description, no matter if it is not that mature.

Reply:
it is added dynamically.

Yes agree, thats why TODO Description

Reply:
r u a wiz?